OpenRA.Mods.Common/Activities/Move/AttackMoveActivity.cs

```diff
@@ -51,6 +93,9 @@ public override bool Cancel(Actor self, bool keepQueue = false)
 			if (!IsCanceled && inner != null && !inner.Cancel(self))
 				return false;
 
+			if (!IsCanceled && attack != null && !attack.Cancel(self))
+				return false;
+
 			return base.Cancel(self, keepQueue);
 		}
 
```

Comment:
`GetTargets` below will want to be updated to return either `target`'s target or `inner`'s target if `target` is null, I think.

Reply:
Handing off to `inner` would be better, otherwise it will likely cause problems when fixing #12019.

Reply:
Okay, then I'll keep it as is.